clang/test/CIR/CodeGen/CUDA/simple.cu

```diff
@@ -1,9 +1,11 @@
 #include "../Inputs/cuda.h"
 
-// RUN: %clang_cc1 -triple nvptx64-nvidia-cuda -fclangir \
-// RUN:            -emit-cir %s -o %t.cir
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir \
+// RUN:            -x cuda -emit-cir %s -o %t.cir
 // RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
 
+// Attribute for global_fn
+// CIR: #cir.cuda_kernel_name<_Z9global_fnv>
```

Comment:
You need to check this in the function signature as well, you can use FileCheck's regexes to tie them together: https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-regex-matching-syntax